src/lib/provas/2019/19.json

```
@@ -0,0 +1,18 @@
+{
+    "ano": "2019",
+    "questao": "19",
+    "area": "Matemática",
+    "enunciado": "– De quantas maneiras diferentes é possível formar uma equipe de ginástica olímpica \ncom precisamente 3 ginastas mulheres e 4 ginastas homens, escolhidos a partir de uma delegação \ncom 15 pessoas, das quais 8 são homens e 7 são mulheres?",
```

O travessão no começo do enunciado foi proposital?
```suggestion
    "enunciado": "De quantas maneiras diferentes é possível formar uma equipe de ginástica olímpica \ncom precisamente 3 ginastas mulheres e 4 ginastas homens, escolhidos a partir de uma delegação \ncom 15 pessoas, das quais 8 são homens e 7 são mulheres?",
```

---

Alem disso, pode-se remover os "\n" também.

```suggestion
    "enunciado": "– De quantas maneiras diferentes é possível formar uma equipe de ginástica olímpica com precisamente 3 ginastas mulheres e 4 ginastas homens, escolhidos a partir de uma delegação com 15 pessoas, das quais 8 são homens e 7 são mulheres?",
```